src/VehicleSetup/JoystickConfigAdvanced.qml

```diff
@@ -88,6 +88,23 @@ Item {
             }
         }
         //-----------------------------------------------------------------
+        //-- Zero throttle while not flying
+        QGCLabel {
+            text:               qsTr("Zero throttle while aircraft not flying")
+            Layout.alignment:   Qt.AlignVCenter
+            Layout.minimumWidth: ScreenTools.defaultFontPixelWidth * 36
+        }
+        QGCCheckBox {
+            id:         zeroThrottleNotFlying
+            checked:    activeVehicle.joystickMode !== 0
```

Comment:
So this should always be checked if joystick mode is not `JoystickModeRC`?

Reply:
Removed the line.